src/main/java/org/cactoos/io/LoggingInputStream.java

```diff
@@ -23,6 +23,7 @@
  */
 package org.cactoos.io;
 
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
```

Comment:
@ilyakharlamov the same above.

Reply:
The root cause is fixed in 7cece16 so there will be no need for the supression.

Reply:
@ilyakharlamov so why do we still have the suppression?